### PR TITLE
Prepare 2.0.0 release

### PR DESCRIPTION
**Pull request by jillr**
##### SUMMARY
Prepare 2.0.0 release.  Run add_docs, generate changelog, update versions

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
amazon.aws

### CODE REVIEW

**Comment by tremble** [comment]
Looks like there might be something wrong with the ec2_eni_info markup?

```
2021-09-03 22:33:13.906380 | centos-8-stream | ERROR: docs/amazon.aws.ec2_eni_info_module.rst:481:0: Explicit markup ends without a blank line; unexpected unindent.
```

**Comment by jillr** [comment]
> Looks like there might be something wrong with the ec2_eni_info markup?
> 
> ```
> 2021-09-03 22:33:13.906380 | centos-8-stream | ERROR: docs/amazon.aws.ec2_eni_info_module.rst:481:0: Explicit markup ends without a blank line; unexpected unindent.
> ```

Yeah add_docs.py does something funky and I just haven't taken the time to figure it out.  I just manually add the extra indentations.

**Comment by tremble** [comment]
> Yeah add_docs.py does something funky and I just haven't taken the time to figure it out. I just manually add the extra indentations.

could be an add_docs bug, ec2_eni_info looks like the only amazon.aws module where version_added is used in RETURN

**Comment by jillr** [comment]
I think ec2_spot_request_info will take a few more days to merge and I don't want to block the release. If it merges fast I'll do a quick refresh of this PR.

ready_for_review

**Comment by tremble** [comment]
@jillr The one remaining change I'd like to see prior to 2.0.0 is #488 it's got the various approvals and just needs gating but given the scope I'd prefer a member of your team do the final gating.

**Comment by jillr** [comment]
> @jillr The one remaining change I'd like to see prior to 2.0.0 is #488 it's got the various approvals and just needs gating but given the scope I'd prefer a member of your team do the final gating.

@tremble I'm fine with that if we can get it through.  I haven't had a chance to dig into the CI failures, is it just the ec2_metadata change you PRed?

**Comment by tremble** [comment]
> I'm fine with that if we can get it through. I haven't had a chance to dig into the CI failures, is it just the ec2_metadata change you PRed?

Yeah the metadata PR and some timeouts which (as far as I can tell) are cause by too many things in one bucket.

**Comment by alinabuzachis** [comment]
@jillr FYI
```
ERROR: Found 1 rstcheck issue(s) which need to be resolved:
ERROR: docs/amazon.aws.ec2_eni_info_module.rst:481:0: Explicit markup ends without a blank line; unexpected unindent.
See documentation for help: https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/rstcheck.html
```

**Comment by goneri** [comment]
recheck

**Review by tremble — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst
Comment (jillr) [reply CHANGELOG.rst:15]: I must have missed it because --refresh-fragments is throwing an error.  I'll get that sorted asap

Comment (goneri) [reply CHANGELOG.rst:15]: It would be nice if `antsibull-changelog` was able to regenerate the last entry if we call it a second time with fixed changelog fragment.

File: CHANGELOG.rst
Comment (tremble) [inline CHANGELOG.rst:14]:
```suggestion
```

manually remove this line?

**Review by tremble — APPROVED** [review]

**Pull request merged**
